cypress/e2e/orders/transactionsForOrders.js

```diff
@@ -136,37 +135,38 @@ describe("Orders", () => {
     },
   );
 
-  it(
-    "should be able to grant and send refund TC: 3902",
-    { tags: ["@orders", "@allEnv", "@stable"] },
-    () => {
-      createReadyToFulfillOrder({
-        customerId: customer.id,
-        shippingMethod,
-        channelId: channel.id,
-        variantsList,
-        address,
-      }).then(({ order: orderResp }) => {
-        const orderPrice = orderResp.total.gross.amount;
+  // TODO uncomment when bug: https://github.com/saleor/saleor/issues/12757 if fixed
```

Comment:
**question**: shouldn't this test be removed?

If not maybe you can skip it instead of commenting out?

Reply:
I agree we can use it.skip or xit till the bug will be fixed